website/versioned_docs/version-25.x/MockFunctions.md

```diff
@@ -17,41 +17,49 @@ function forEach(items, callback) {
     callback(items[index]);
   }
 }
+module.exports = forEach;
 ```
 
 To test this function, we can use a mock function, and inspect the mock's state to ensure the callback is invoked as expected.
 
-```javascript
+```js title="foreach.test.js"
+const forEach = require('./foreach');
+
 const mockCallback = jest.fn(x => 42 + x);
-forEach([0, 1], mockCallback);
 
-// The mock function is called twice
-expect(mockCallback.mock.calls.length).toBe(2);
+test('forEach mock function', () => {
+  forEach([0, 1], mockCallback);
 
-// The first argument of the first call to the function was 0
-expect(mockCallback.mock.calls[0][0]).toBe(0);
+  // The mock function was called twice
+  expect(mockCallback.mock.calls).toHaveLength(2);
```

Comment:
Could you use `.toHaveLength` in this line too:

https://github.com/facebook/jest/blob/41bf2300895a2c00d0525d21260f0a392819871f/docs/MockFunctions.md?plain=1#L63